community/halloffame.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: community
+title: Hall of Fame
+---
+
+The Hall of Fame highlights occasions where the Scala.js project has contributed to other, larger projects.
+
+### Node.js/V8: Integer Arithmetic Bug
+
+Prior to Node.js 0.10.29, comparing `Int.MaxValue` with `Int.MinValue` could yield a wrong result.
+Notably:
+
+{% highlight javascript %}
+$ node
+> 2147483647 > -2147483648
+false
+{% endhighlight %}
+
+This [Node.js bug][1] was discovered by a [test][2] in Scala's partest suite that we run for Scala.js.
```

Comment:
Hahaha, ok. I used this notation so you can read the .md reasonably well :P I'll change it.